src/c_frontend/tdi_table_c.cpp

```diff
@@ -629,12 +514,11 @@ tdi_status_t tdi_action_id_applicable(const tdi_table_hdl *table_hdl,
 }
 
 #ifdef __TDI_FROM_BFRT
```

Comment:
If we enable the tdi_table_attributes_set, then we need to remove the __TDI_FROM_BFRT.